afterSignHook.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const { notarize } = require('electron-notarize')
+
+/**
+ *
+ * @param {*} context - Object containint electron-builder context
+ * @returns {Promise}
+ *
+ * @example
+ * await notarize({
+ *   // electron-builer context
+ * })
+ */
+const notarizing = async (context) => {
+  const { electronPlatformName, appOutDir } = context
+  if (electronPlatformName !== 'darwin') {
+    return Promise.resolve()
```

Comment:
You can simply `return` here, no need for `Promise.resolve()`, this is already an async function.

Reply:
The linter complains about the function returning void then a Promise otherwise

Reply:
The linter is old and wrong, disable it for this file.